frontend/src/components/table/schedule/schedule-parts/base-cell.component.tsx

```diff
@@ -43,7 +65,28 @@ export function BaseCellComponent({
       }}
     >
       {isPointerOn && !isBlocked && (
-        <input defaultValue={value} autoFocus={true} ref={inputRef} className="cell-input" />
+        <div>
+          <TextField value={value} onClick={handleClick} />
+          <Popper open={open} anchorEl={anchorEl}>
+            <Autocomplete
+              id="combo-box-demo"
+              className="cell-input"
+              autoSelect={true}
+              options={ShiftCodeTranslation}
+              getOptionLabel={(option) => option.symbol}
+              style={{ width: 200 }}
+              renderInput={(params) => <TextField {...params}
+                style={{
+                  backgroundColor: style.backgroundColor.toString()
+                }}
+                label="Przypisz zmianę"
+                variant="outlined"
+                value={(option) => option.symbol} />}
+              renderOption={(option) => option.name}
+              ref={inputRef}
```

Comment:
Zgodnie z dokumentacją tutaj przez parametr onChange wstawiamy funkcję (event, value) => {} i value to nasze nowe wybrane value, aby je zapisać niestety nie będzie to takie proste, bo działało to wcześniej przy kliknięciu ENTER. @Qwebeck to może być duża zmiana w wielu miejscach, z tego co na szybko przeglądałem. Żeby dostarczyć możliwość aktualizacji stanu na zwykłym callbacku z value bez eventa.

Reply:
Teraz spróbuję to rozwiązać

Reply:
Zmieniłem api w taki sposób, że teraz komórka przyjmuje metodę `onValueChange`, która jest wywoływana w momencie zmiany zawartości komórki z dropdowna .